lib/index.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
   inspect: inspect,
 };
 
+var VIRTUAL_ROOT_NODE_ID = '.'
```

Comment:
const?

Reply:
cant, its should support node 0.12

Reply:
actually it does work on node 0.12, but it only partly supports the standard: https://node.green/#ES2015-bindings-const, so maybe that's the reason there is no use of `const` in our CLI plugins and `snyk-internal` itself